### PR TITLE
feat : add signup page

### PR DESCRIPTION
**Pull request by viru2001**
## created signup page

image

[Deploy Preview](https://deploy-preview-3--gamers-bazaar.netlify.app/pages/signup.html)

### CODE REVIEW

File: pages/signup.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Gamers Bazaar | Sign Up</title>
+
```
Comment (tushargoyal812) [inline pages/signup.html:8]: suggestion: remove unnecessary space

**Review by tushargoyal812 — COMMENTED** [review]
only take care what i suggested,looks good to merge

File: pages/signup.html (hunk 1 above)
Comment (viru2001) [reply pages/signup.html:8]: I have added that space intentionally.

**Review by saifudinbadani — APPROVED** [review]
looks good to merge.

**Pull request merged**